### PR TITLE
Mjor: Creator of Original Assets

### PR DESCRIPTION
**Pull request by MMMiracles**
:cl: MMMiracles
add: A giant wad of paper has been seen hurling through the nearby sector of SS13, officials deny existence of giant paper ball hurling through space because 'thats really fucking stupid'.
/:cl:

paperba

Mjor the Creative, a wizard with the taste for the arts, has used his magical craft to form a home made of papier-mâché hurling through space because hes a wizard he can do whatever the fuck he wants. He uses his 'artistic' skill to conjure up minions to assist him when nosy people try to steal his _priceless_ art.

He has two abilities he can use during a fight, one will summon copies of himself as mimics. Attacking a mimic does 50 brute to everyone around him and attacking him destroys all active mimics. The other ability lets him summon his crudely-drawn creations, bringing them to life to fight for him. 

Killing him gives you access to his papier-mâché robe, a special collector's edition paper, and his _priceless_ range of artwork. What more could you ask for?
painting1
painting2
painting3

The wizard's robe, when equipped, will allow the wearer to summon their own stickman allies. They are only loyal to who summoned them and will attack everything else. Ability has a cooldown between each use.

Boss code credit majority to @RemieRichards as it is a modified version of what she wrote up a couple months ago for one of my unfinished away missions.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
ignoring that commit title, this is good :+1:

File: code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+//fancy effects
+/obj/effect/paper_scatter
+	name = "scattering paper"
+	desc = "Pieces of paper scattering to the wind."
+	layer = ABOVE_OPEN_TURF_LAYER
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "paper_scatter"
+	anchored = TRUE
+
+/obj/effect/paper_scatter/New()
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm]: we have a ```/obj/effect/timed``` or something to that effect, that handles deletion after x seconds and such.

File: code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+	desc = "A wormhole sucking the wizard into the void. Neat."
+	layer = ABOVE_OPEN_TURF_LAYER
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "paperwiz_poof"
+	anchored = TRUE
+
+/obj/effect/paperwiz_dying/New()
+	..()
+	visible_message("<span class='boldannounce'>The wizard cries out in pain as a gate appears behind him, sucking him in!</span>")
+	playsound(get_turf(src),'sound/magic/MandSwap.ogg', 50, 1, 1)
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm]: MandSwap?

Comment (Bawhoppen) [inline code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm]: MANSWAP COMING SOON

**Comment by RemieRichards** [comment]
That commit tho.
What's your favourite idea?

File: code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm (hunk 2 above)
Comment (MMMiracles) [inline code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm]: That's the name of the sound used for mindswap, no idea why but it works.

File: code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+		..()
+
+//Hit the real guy? copies go bai-bai
+/mob/living/simple_animal/hostile/boss/paper_wizard/adjustHealth(amount)
+	if(amount > 0)//damage
+		minimum_distance = 3
+		retreat_distance = 3
+		for(var/copy in copies)
+			qdel(copy)
+			..()
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm]: this should be . = ..() and also have two less tabs so it doesn't happen once per copy and actually permits admin-forced healing

File: code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm (hunk 1 above)
Comment (MMMiracles) [inline code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm]: As far as I can tell, there isn't actually anything like that.

Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm]: ```/obj/effect/overlay/temp```?

Comment (MMMiracles) [inline code/modules/mob/living/simple_animal/hostile/bosses/paperwizard.dm]: I was looking for `/obj/effect/temp` awh geez

**Comment by KorPhaeron** [comment]
Damn, was too slow to use the mimic code for a lavaland mob

**Comment by KorPhaeron** [comment]
and we really need a way for people to more properly explore space, it's a shame this stuff will almost never be seen. Looks really cool.

**Comment by Incoming5643** [comment]
@KorPhaeron Blame #19276

**Comment by RemieRichards** [comment]
I think he means more like more methods of space travel.
Pods and such.

**Comment by jughu** [comment]
1+ is there a chance the robe gets added to the wizards spell selection?

**Comment by RemieRichards** [comment]
@jughu ```:+1:```

**Comment by lordpidey** [comment]
Throw a couple of pens and toner refills on the floor.

**Comment by TheWiznard** [comment]
I was expecting doodlebob

**Comment by Aloraydrel** [comment]
Should make paper airlocks to fit the theme. Plasma ones seem pretty stuck out

**Comment by MMMiracles** [comment]
I did some map edits to make the airlocks fit better aesthetically, as well as making the robe drop allow the wearer to summon their own stupid looking stick minions.

File: _maps/RandomRuins/SpaceRuins/originalcontent.dmm [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+"cu" = (/obj/structure/fluff/paper{tag = "icon-paper (WEST)"; icon_state = "paper"; dir = 8},/obj/structure/fluff/paper/stack{tag = "icon-paperstack (SOUTHEAST)"; icon_state = "paperstack"; dir = 6},/turf/open/indestructible{desc = "A floor made of invulernable notebook paper."; icon_state = "paperfloor"; name = "notebook floor"},/area/ruin/powered)
+"cv" = (/obj/structure/fluff/paper,/obj/structure/fluff/paper/stack{tag = "icon-paperstack (SOUTHEAST)"; icon_state = "paperstack"; dir = 6},/turf/open/indestructible{desc = "A floor made of invulernable notebook paper."; icon_state = "paperfloor"; name = "notebook floor"},/area/ruin/powered)
+"cw" = (/obj/structure/fluff/paper{tag = "icon-paper (NORTH)"; icon_state = "paper"; dir = 1},/mob/living/simple_animal/hostile/stickman/ranged,/turf/open/indestructible{desc = "A floor made of invulernable notebook paper."; icon_state = "paperfloor"; name = "notebook floor"},/area/ruin/powered)
```
Comment (xxalpha) [inline _maps/RandomRuins/SpaceRuins/originalcontent.dmm]: Denied

**Comment by KorPhaeron** [comment]
I recommend adding the invisible tiny fans to the exits if you have not already

**Comment by PKPenguin321** [comment]
we really really really need a ruin that's a "Massive Tear in Space-Time" that has all the original assets from the Exadv1 builds

**Comment by jughu** [comment]
Only if flamethrowers smelts the whole paper thing

**Comment by RemieRichards** [comment]
@PKPenguin321 we have an oldstation ruin with old sprites

File: _maps/RandomRuins/SpaceRuins/originalcontent.dmm (hunk 4 above)
Comment (MMMiracles) [inline _maps/RandomRuins/SpaceRuins/originalcontent.dmm]: ill deny you

**Comment by PKPenguin321** [comment]
whaaaaaat

File: _maps/RandomRuins/SpaceRuins/originalcontent.dmm (hunk 4 above)
Comment (optimumtact) [inline _maps/RandomRuins/SpaceRuins/originalcontent.dmm]: make them into a turftype thank you.

**Comment by Iamgoofball** [comment]
@RemieRichards we dont but I'm adding one

**Comment by RemieRichards** [comment]
@Iamgoofball we should do tho? HG wanted me to use them for VR, and someone joined the convo to explain we already have an oldstation ruin.
Which I also have faint memories of.

**Comment by KorPhaeron** [comment]
We have an oldstation shuttle ruin

**Comment by Iamgoofball** [comment]
wow, rip

**Comment by MMMiracles** [comment]
I did the oldstation shuttle ruin awhile ago, it even has the shitty old spacemen inside it.

Anyway quick merge it I'm sure travis will pass with flying colors no problem here.

**Comment by Bawhoppen** [comment]
this pr fucking deleted some sprites inside floors.dmi thanks a fucking lot MMMiracles

**Comment by Incoming5643** [comment]
A strong reminder: The way to resolve conflicts in dmi files is to open both versions of the file and move the new sprites from your version into their version, then using their version

It's not to just use your version and pray everything will still work out

**Comment by MMMiracles** [comment]
You see, I meant to add the additional icons after fixing the conflicts, but I just never actually did.

Whoops.
¯\_(ツ)_/¯

**Pull request merged**